src/admin-views/aggregator/settings.php

```diff
@@ -364,6 +364,38 @@
 
 $internal = apply_filters( 'tribe_aggregator_fields', $internal );
 
+// fix this shit
+
+
+
+if ( Tribe__Events__Aggregator::instance()->is_service_active() ) {
+	$import_instructions = __( 'Use the options below to configure your imports. Global Import Settings apply to all imports, but you can also override the global settings by adjusting the origin-specific options.', 'the-events-calendar' );
+	$import_instructions .= '<br><br>';
```

Comment:
Could be more readable to simply concatenate the strings and ditch the variable name from the start of all but the first line (ie, instead of `$var_name .= 'more text';` use `. 'more text'` for each line).

Also, could there be a better, perhaps CSS based option than using `<br><br>`? If we need to get this out the door for reasons of timeliness and come back to address later that's fine.